lib/package_json.ts

```diff
@@ -112,6 +112,7 @@ export function getPackageJson({
     ...mainExport,
     ...binaryExport,
     ...packageJsonObj,
+    scripts: {},
```

Comment:
this change is needed in addition, as the splat from `deleteEmptyKeys` won't have the `scripts` key if it is an empty object.